test/exclude-members.test.js

```diff
@@ -160,6 +145,32 @@ test('add and remove a person, post on the new feed', async (t) => {
 
   t.equal(post.text, 'post', 'found post on second feed')
 
+  const aliceAdditions = await p(alice.metafeeds.findOrCreate)({
+    purpose: 'group/additions',
+  })
+  const msgsFromAdditions = await alice.db.query(
+    where(author(aliceAdditions.id)),
+    toPromise()
+  )
```

Comment:
:pineapple: it's cool seeing different ways of querying around the space + testing. I would have got a different way but this is tidy (looking within a particular sub-feed).
db.query is so sweet